source/common/upstream/cluster_manager_impl.h

```diff
@@ -326,6 +326,13 @@ class ClusterManagerImpl : public ClusterManager,
   initializeSecondaryClusters(const envoy::config::bootstrap::v3::Bootstrap& bootstrap) override;
 
   const ClusterStatNames& clusterStatNames() const override { return cluster_stat_names_; }
```

Comment:
do we still use ClusterStatNames anywhere?

Reply:
I am not sure. 
I saw it only used to instantiate the ClusterStats in upstream_impl.cc.

Reply:
can you comment it out and see what breaks?

I'd like this PR to just be a repartitioning without adding any overlap.